features/fields_extensie.feature

```diff
@@ -1,126 +1,544 @@
+      Dan bevat de response de volgende gegevens
+      | naam     | waarde                                                                                                      |
+      | type     | https://docs.microsoft.com/en-us/dotnet/api/system.net.httpstatuscode?#System_Net_HttpStatusCode_BadRequest |
+      | title    | Een of meerdere parameters zijn niet correct.                                                               |
+      | status   | 400                                                                                                         |
+      | detail   | De foutieve parameter(s) zijn: fields.                                                                      |
+      | code     | paramsValidation                                                                                            |
+      | instance | /personen                                                                                                   |
+      En bevat de response een invalidParams met de volgende gegevens
+      | code     | name          | reason                        |
+      | required | fields        | Parameter mag niet leeg zijn. |
+
+    Scenario: Fields parameter met attribuutnaam die niet bestaat
+      Gegeven het systeem heeft een persoon met de volgende gegevens
+      | naam                | waarde    |
+      | burgerservicenummer | 999994086 |
+      | geslachtsaanduiding | M         |
+      Als personen wordt gezocht met de volgende parameters
+      | naam                | waarde                                  |
+      | type                | RaadpleegMetBurgerservicenummer         |
+      | burgerservicenummer | 999994086                               |
+      | fields              | burgerservicenummer, bestaatniet        |
+      Dan bevat de response de volgende gegevens
+      | naam     | waarde                                                                                                      |
+      | type     | https://docs.microsoft.com/en-us/dotnet/api/system.net.httpstatuscode?#System_Net_HttpStatusCode_BadRequest |
+      | title    | Een of meerdere parameters zijn niet correct.                                                               |
+      | status   | 400                                                                                                         |
+      | detail   | De foutieve parameter(s) zijn: fields.                                                                      |
+      | code     | paramsValidation                                                                                            |
+      | instance | /personen                                                                                                   |
+      En bevat de response een invalidParams met de volgende gegevens
+      | code     | name          | reason                                           |
+      | required | fields        | Parameter bevat een niet bestaande propertynaam. |
```

Comment:
Code is dan fields (althans in v1.3 was dat zo)

Reply:
Aangepast.